examples/multiplayer-netfox/characters/player.tscn

```diff
@@ -22,7 +22,6 @@ script = ExtResource("2_qf1b4")
 root = NodePath("..")
 state_properties = Array[String]([":position"])
 input_properties = Array[String](["Input:movement"])
-interpolate_properties = Array[String]([":position"])
```

Comment:
Originally, RollbackSynchronizer managed interpolation as well. We would need to add a *TickInterpolator* node with the `:position` property configured.